gui/src/components/alerts/alert-details.vue

```diff
@@ -229,29 +229,13 @@ export default class CortxAlertDetails extends Vue {
       this.alert = res.data;
       try {
         if (this.alert.extended_info) {
```

Comment:
if (this.alert.extended_info) {
// Can You please add an additional check as 
 if (this.alert && this.alert.extended_info) {

//Same for event_details

Reply:
This is already done two lines above. if(res.data){  this.alert = res.data;}

Reply:
@shrimetta  is this change required in CORTX1.0 too.. I think yes.. Please update that code and let me know